variables.tf

```diff
@@ -105,3 +105,10 @@ variable "route_only_private_ips" {
   description = "Routes only private IPs through the VPN (10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16)"
   default     = false
 }
+
+variable "openvpn_install_commit_sha" {
```

Comment:
rename this variable to `install_script_commit_sha`. Add the link to the script in the description

Reply:
Okay I have done these changes locally

Reply:
Have pushed the changes